core/js/src/Instant.kt

```diff
@@ -75,12 +75,7 @@ public actual class Instant internal constructor(internal val value: jtInstant)
             if (epochMilliseconds > 0) MAX else MIN
         }
 
-        actual fun parse(isoString: String): Instant = try {
-            Instant(jtInstant.parse(isoString))
-        } catch (e: Throwable) {
-            if (e.isJodaDateTimeParseException()) throw DateTimeFormatException(e)
-            throw e
-        }
+        actual fun parse(isoString: String): Instant = parseInstantCommon(isoString)
```

Comment:
This could be implemented easily—and consistently with the other parsers—by parsing a `ZonedDateTime` and then converting that to `Instant`.

The same for the Java implementation.